### DISCUSSION TITLE
Upload Speed limited?

### DISCUSSION THREAD

**Discussion by naddel91** [rustdesk/rustdesk#9250] [General]
Hi at all,

I am on a fibre connection andwhen I transfer larger files the other client only receives 16 mbit/s (also fibre connection). why is that? it could be way faster. is there a limit in rustdesk or something I can improve to make it faster?
best wishes

**Comment by basilgello** [comment]
Do you connect via public relay servers?

**Comment by open-trade** [comment]
Also limited by the implementation, but this is open source, PR is welcome to improve the code.

**Comment by xlionjuan** [comment]
Change `SINGLE_BANDWIDTH`

https://github.com/rustdesk/rustdesk-server?tab=readme-ov-file#env-variables
